### PR TITLE
Replicated build process

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

**Comment by bcoe** [comment]
\o/ :shipit:

File: replicated/start.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+chown -R couchdb:couchdb /etc/npme/couchdb
+cd /etc/npme
```
Comment (emosbaugh) [inline replicated/start.sh]: dont need this. already have `WORKDIR /etc/npme`

File: replicated/Dockerfile.replicated [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Expose ports
+EXPOSE 8080
+
+# Set the default directory where CMD will execute
+WORKDIR /etc/npme
+
+COPY start.sh /start.sh 
+
+# CMD service redis-server start | service nginx start | couchdb | npme restart | tail -f ./logs/*
+ENTRYPOINT ["/start.sh"]
```
Comment (emosbaugh) [inline replicated/Dockerfile.replicated]: you probably want CMD here right? otherwise idk if admin commands will work

**Comment by bcoe** [comment]
this has gone live.

**Pull request merged**